cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/GetController.java

```diff
@@ -21,35 +19,36 @@
 import java.util.Map;
 import java.util.concurrent.Callable;
 
-
 /**
- * This class realizes the functionality that is needed for the get commands. They provide various information about a
- * cloud foundry instance.
+ * This class realizes the functionality that is needed for the get commands.
+ * They provide various information about a cloud foundry instance.
  */
-@Command(name = "get",
-        header = "%n@|green Get the current configuration of your cf instance.|@",
-        mixinStandardHelpOptions = true,
-        subcommands = {
-                GetController.GetServicesCommand.class,
-                GetController.GetSpaceDevelopersCommand.class,
-                GetController.GetApplicationsCommand.class,
-                GetController.GetAllInformation.class})
+@Command(name = "get", header = "%n@|green Get the current configuration of your cf instance.|@",
+    mixinStandardHelpOptions = true,
+    subcommands = {
+    GetController.GetServicesCommand.class,
+    GetController.GetSpaceDevelopersCommand.class,
+    GetController.GetApplicationsCommand.class
+})
```

Comment:
We should not remove the GetAllInformation class because this is not desired in the user story.